Cargo.toml

```diff
@@ -20,7 +20,7 @@ default = ["native-tls"]
 [dependencies]
 native-tls = { version = "0.2.2", optional = true }
 rustls-connector = { version = "0.16.1", optional = true }
-regex = "1.0"
+regex = "1.5"
```

Comment:
This feels unnecessary. We do not depend on anything from 1.5 (to my knowledge). We should leave bumping for security fixes and the like to anything that actually builds binaries and locks dependencies, which we do not.

Reply:
fair enough..  This was just based on the dependency checker you have linked in the README ( https://deps.rs/repo/github/jonhoo/rust-imap ). So to make that happy we should do `>= 1.5.5` OR just do `1.6`. But if you do not care then I'll revert this line.

Reply:
Yeah, I'm generally opposed to libraries dictating a higher minimum version than they technically require, and think deps.rs is over-zealous here.